### PR TITLE
community/mpv: split development files into mpv-dev

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: testing/vidcutter/APKBUILD [hunk 1]
```diff
@@ -7,7 +7,7 @@ pkgdesc="A modern, simple to use, constantly evolving and hella fast MEDIA CUTTE
 url="https://vidcutter.ozmartians.com"
 arch="all"
 license="GPL-3"
-depends="python3 ffmpeg mediainfo mpv py3-qt5"
+depends="python3 ffmpeg mediainfo mpv-dev py3-qt5"
```
Comment (ncopa) [inline testing/vidcutter/APKBUILD]: are the mpv development files needed for vidcutter runtime? wouldnt it be enought to add `mpv-dev` to `makedepends`?

Comment (maxice8) [reply testing/vidcutter/APKBUILD]: I guess not, i'll check.

Comment (maxice8) [reply testing/vidcutter/APKBUILD]: well the package is broken. it needs py3-opengl , py3-setuptools

**Comment by maxice8** [comment]
@ncopa i kept mpv on vidcutter but it doesn't matter much since it needs py3-opengl (unpackaged) and py3-setuptools in depends.

**Comment by maxice8** [comment]
ok with #7226 vidcutter seems to work.

**Pull request closed without merging**